frontend/components/FormLayout.vue

```diff
@@ -0,0 +1,41 @@
+<template>
```

Comment:
I really like this solution! I could see it be useful for other "small form" pages and modals in the future